.travis.yml

```diff
@@ -70,67 +77,326 @@ jobs:
+    - language: java
+      jdk: openjdk7
+      env: DISABLE_BEARS=tailor
+
+    - language: dart
+      dart: 1.14.2
+
+    - language: julia
+      julia: 1.0
+      env: JL_PKG=CoalaBears
+      install:
+        - julia --color=yes .ci/deps.julia.jl
+        - julia -e 'import Lint.lintfile'
+
+    - language: julia
+      julia: 0.7.0
+      env: JL_PKG=CoalaBears
+      install:
+        - julia --color=yes .ci/deps.julia.jl
+        - julia -e 'import Lint.lintfile'
+
+    - language: r
+      r: devel
+
+    - language: r
+      r: oldrel
+
+    - language: generic
+      env: DIST=trusty BEARS=apt_get
+      addons:
+        apt:
+          packages:
+            - chktex
+            - cppcheck
+            - devscripts
+            - flawfinder
+            - indent
+            - libperl-critic-perl
+            - libxml2-utils
+            - mono-mcs
+            - php5-cli
+            - php-codesniffer
+            - shellcheck
+            - verilator
+
+    - stage: test-apt
+      language: generic
+      env: DIST=precise BEARS=apt_get
+      dist: precise
+      addons:
+        apt:
+          packages:
+            - chktex
+            - cppcheck
+            - devscripts
+            - flawfinder
+            - indent
+            - libperl-critic-perl
+            - libxml2-utils
+            - mono-mcs
+            - php5-cli
+            - python3
+            - verilator
+
+    - stage: test-master
+
+      # This is under other languages, as the only bear is tailor
+      # which is java, already covered above.  Listed first to schedule
+      # it early onto osx, as they take a while to spin up
+      language: objective_c
+      os: osx
+      dist: false
+
     - *check_moban
   allow_failures:
     - *check_moban
 
-dist: trusty
-
-.apt_sources: &apt_sources
-  - ubuntu-toolchain-r-test
-  # avsm  # OPAM stable
-  - hvr-ghc  # Haskell
-  - sourceline:  # R
-      deb https://cloud.r-project.org/bin/linux/ubuntu trusty-cran35/
-    key_url:
-      https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x51716619E084DAB9
-  - sourceline:  # Julia
-      deb http://ppa.launchpad.net/staticfloat/juliareleases/ubuntu trusty main
-    key_url:
-      https://keyserver.ubuntu.com/pks/lookup?op=get&search=0xCF979FFA3D3D3ACC
-  - sourceline:  # astyle
-      deb http://ppa.launchpad.net/cs50/ppa/ubuntu trusty main
-    key_url:
-      https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x5BDA2E974A0E822C
-
-addons:
-  apt:
-    sources: *apt_sources
-    packages:
-      - aspcud
-      - astyle
-      - cabal-install-1.24
-      - chktex
-      - clang-3.4
-      - cppcheck
-      - devscripts
-      - flawfinder
-      - gfortran
-      - ghc
-      - happy
-      - indent
-      - julia
-      - libarpack2
-      - libblas-dev
-      - libcolamd2.8.0
-      - libfftw3-3
-      - liblapack-dev
-      - libopenblas-base
-      - libpaper-utils
-      - libperl-critic-perl
-      - libumfpack5.6.2
```

Comment:
need to get out the blame for these libraries -- they probably made one of the languages faster; probably R

Reply:
64e95b30

Re-adding colamd didnt cause it to not be built.  https://travis-ci.org/jayvdb/coala-bears/builds/559524918

Probably better to try getting binary packages working after this PR.